### PR TITLE
LPS-192949 Deletion Modification Conflict for DDM Structures should not be edited

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPS-192949](https://liferay.atlassian.net/browse/LPS-192949)

### CODE REVIEW

**Comment by david-truong** [comment]
@noornajj can you review this one?

**Comment by noornajj** [comment]
Hi @gislayne-vitorino 

The changes look good but for the if condition I don't think you need to check for ddmStructure.
I was able to reproduce this same issue with page templates and that shouldn't have an option to edit either. I believe you can have the condition just check whether it's a deletion modification conflict. I think we would want to prevent this action for all these types of conflicts. Is that right @david-truong?

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by noornajj** [comment]
@gislayne-vitorino 

This looks good to me. Thanks!

cc @david-truong

File: modules/apps/change-tracking/change-tracking-web/build.gradle [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependencies {
 	compileOnly project(":apps:change-tracking:change-tracking-spi")
 	compileOnly project(":apps:configuration-admin:configuration-admin-api")
 	compileOnly project(":apps:diff:diff-api")
+	compileOnly project(":apps:dynamic-data-mapping:dynamic-data-mapping-api")
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/build.gradle:18]: @gislayne-vitorino is this dependency needed?

Comment (gislayne-vitorino) [reply modules/apps/change-tracking/change-tracking-web/build.gradle:18]: No, looks like it's not needed anymore. will send an update now

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
@xbrianlee is this safe to forward?

**Comment by xbrianlee** [comment]
ci:test:publications

Re-running this suite since there were some aborted tests in the last run. `relevant` results look good. We should be able to forward as long as the publications results look good.

**Comment by suilin** [comment]
ci:forward:force

**Pull request closed without merging**